JASP-Engine/JASP/po/R-nl.po

```diff
@@ -915,31 +1277,98 @@ msgid "Audit values"
-"audit values. Red dots indicate observations that did not match their "
-"original book value. If these red dots lie in the bottom part of the graph, "
-"the book values are overstated. If these red dots lie in the upper part of "
-"the graph, they are understated. The value <i>r</i> is the Pearson "
-"correlation coefficient of the book values and the audit values, an "
-"indicator of the strength of the linear relationship between the two "
-"variables."
+"<b>Figure 1.</b> Scatter plot of the book values in the selection and their "
+"audit values. Red dots indicate observations that did not match their "
+"original book value. If these red dots lie in the bottom part of the graph, "
+"the book values are overstated. If these red dots lie in the upper part of "
+"the graph, they are understated. The value <i>r</i> is the Pearson "
+"correlation coefficient of the book values and the audit values, an "
+"indicator of the strength of the linear relationship between the two "
+"variables."
+msgstr ""
+"<b>Figuur 1.</b> Spreidingsgrafiek van de boekwaardes in de selectie en de "
+"bijbehorende auditwaardes. Rode stippen geven observaties aan die niet "
+"overeenkwamen met hun originele boekwaarde. Als deze rode stippen aan de "
+"onderkant van de grafiek liggen, zijn de boekwaardes overschat. Als deze "
+"rode stippen in het bovenste deel van de grafiek liggen, zijn ze onderschat. "
+"De waarde <i>r</i> is de Pearson correlatiecoëfficiënt van de boekwaardes en "
+"de auditwaardes, een indicator van de sterkte van de lineaire relatie tussen "
+"de twee variabelen."
+
+msgid "<b>percentage</b>"
+msgstr "<b>percentage</b>"
+
+msgid "<b>amount</b>"
+msgstr "<b>hoeveelheid</b>"
+
+msgid "<u>Planning</u>"
+msgstr "<u>Planning</u>"
+
+msgid "<u>Selection</u>"
+msgstr "<u>Selectie</u>"
+
+msgid "<u>Evaluation</u>"
+msgstr "<u>Evaluatie</u>"
+
+#, fuzzy
+msgid ""
+"Analysis not possible: Your materiality is higher than, or equal to the "
+"total value of the observations."
+msgstr ""
+"Analyse niet mogelijk: De materialiteit is groter dan de totale waarde van "
+"de observaties."
+
+msgid ""
+"Analysis not possible: Your expected errors are higher than materiality."
+msgstr ""
+"Analyse niet mogelijk: De verwachte fouten zijn groter dan de materialiteit."
+
+#, fuzzy
+msgid "Selection summary"
+msgstr "Selectie grootte"
+
+#, fuzzy
+msgid ""
+"Your sample size is larger than (or equal to) your population size. Cannot "
+"take a sample larger than the population."
+msgstr ""
+"Uw steekproefgrootte is groter dan de populatiegrootte. Kan geen steekproef "
+"nemen groter dan de populatie."
+
+msgid "Your sample size must be larger than 1."
+msgstr ""
+
+msgid ""
+"Your must specify unique record ID's. The row numbers of the data set are "
+"sufficient."
+msgstr ""
+
+#, fuzzy
+msgid "Evaluation summary"
+msgstr "Evaluatie Metriek"
+
+msgid ""
+"Your audit result does not contain only 0's (correct) and 1's (incorrect)."
+msgstr ""
```

Comment:
Uw auditresultaat bevat niet alleen maar 0 (corerct) of 1 (incorrect).

Reply:
Ok